Casks/octave-app.rb

```diff
@@ -1,14 +1,20 @@
 cask 'octave-app' do
-  version '6.1.0'
-  sha256 'fe45f23e307e922e6ad75d3e4ef1b357b3e4d656205f659741986704f96a17f7'
+  version '6.2.0'
+  sha256 'f847a22d386cbb357d996d8a67f03969f9a668d14f512b3890cdd18ec9a2a958'
 
-  url "https://github.com/octave-app/octave-app/releases/download/v6.1.0/Octave-6.1.0.dmg"
+  url "https://github.com/octave-app/octave-app/releases/download/v#{version}/Octave-#{version}.dmg"
   name 'Octave'
   homepage 'https://octave-app.org'
 
-  depends_on macos: '>= :mojave'
-
   auto_updates false
+ 
+  livecheck do
+    url "https://github.com/octave-app/octave-app/releases"
+    strategy :page_match
+    regex(%r{href=.*?/Octave[._-]?(\d+(?:\.\d+)+)\.dmg}i)
+  end
+
+  depends_on macos: '>= :mojave'
 
-  app 'Octave-6.1.0.app'
+  app 'Octave-#{version}.app'
```

Comment:
I'm not sure this works. Does `#{...}` variable interpolation work inside single quotes? When I do a test run against this PR, I get the following error:

```
$ brew reinstall octave-app
==> Downloading https://github.com/octave-app/octave-app/releases/download/v6.2.0/Octave-6.2.0.dmg
==> Downloading from https://github-releases.githubusercontent.com/137706143/c33a9980-9e4d-11eb-9468-ed672fca39ef?X-Amz-Algorithm=
######################################################################## 100.0%
==> Uninstalling Cask octave-app
==> Purging files for version 6.1.0 of Cask octave-app
==> Installing Cask octave-app
==> Purging files for version 6.2.0 of Cask octave-app
Error: It seems the App source '/usr/local/Caskroom/octave-app/6.2.0/Octave-#{version}.app' is not there.
```

Reply:
I fixed it. Thank you for testing